src/components/layout/FooterBar.vue

```diff
@@ -2,14 +2,15 @@
   <footer class="footer">
     <div class="container">
       <div class="content has-text-centered">
-        <p>© 2017-{{ year }} AEGEE - Frontend version {{ pkg.version }}</p>
-        <p>Powered by <a href="//myaegee.atlassian.net/wiki/spaces/GENERAL">Online membership system</a></p>
         <p>
-          <router-link :to="{ name: 'oms.confluence', params: { id: 'terms-of-service' } }">Legal info (simple)</router-link> &bull;
```

Comment:
this one shouldn't be changed if I'm not mistaken, we should use the Confluence one (as we can change it without re-deploying)

Reply:
Actually it's broken for now, but changing the param from id to page_id should fix it I suppose

Reply:
I changed it, but it doesn't work on my machine. It just loads the login page

Reply:
with page_id?

Reply:
yes. It results in the link beeing `/pages/terms-of-service`, which does nothing

Reply:
@serge1peshcoff what should I do with the link?

Reply:
Let's just rename it to have page_id, I'll fix it not loading later (apparently my cors proxy doesn't work as intended, given that we now have redirect from oms-project.atlassian.net to myaegee.atlassian.net)